packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/test/feature_counter/views/counter_page_test.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import 'factories/counter_page_factory.dart';
+
+void main() {
+  group('CounterPage golden tests', () {
+    final deviceBuilders = [
+      generateDeviceBuilder(
+        label: 'counter',
```

Comment:
@GoranPrime, what's the difference between `label` and `Scenario.name`? Can we use the `Scenario.name' instead of label?

Reply:
I thought this way would be more readable, since `label` is used as the file name of the golden, but I've updated it to simply use `Scenario.name` instead